### ISSUE TITLE
Bring back popup video view in call

### ISSUE THREAD

**Issue by lillelilje** [qTox/qTox#2425]
Hello!

After updating recently I found myself faced with a in-chat video view.
I can't find a way to turn on the old "pop-out" video.

Was this purposely removed? Would it be possible to enable this in a setting?

Thank you!

**Comment by agilob** [comment]
Yes, there will be. first things first, improved audio, new av api.

**Comment by lillelilje** [comment]
Alright! Thank you for all the hard work, devs!

**Comment by ovalseven8** [comment]
What are the advantages of the old video view?

**Comment by agilob** [comment]
takes less place on a screen?

tbh I don't see any advantage of the new view.

**Comment by lillelilje** [comment]
I agree agilob. The old video view was small, very flexible and easy to use. Now you have to manually resize all of the components to get something that isn't nearly as neat :)

However, I do see the advantage of the new view. It's more intuative and closer to skype. Should be the default even though I prefer the old one by a long shot.

**Comment by tox-user** [comment]
Is this still needed after https://github.com/qTox/qTox/pull/5047 ?
